### PR TITLE
fix #22321 bug

### LINKED ISSUE #22321: module.portage_config bug with appending accept_keywords
If we run
```yaml
www-servers/apache:
  portage_config.flags:
    - accept_keywords: ['~amd64']
```
then change `~amd64` to `amd64` and run again
then in minion
```
cat portage/package.accept_keywords/www-servers/varnish returns
~amd64 amd64
```
but this two flags couldn't be together - `~amd64` means that we want to install the last unstable version, and `amd64` flags means that we want to install stable version.
I afraid, that this issue could be avoid, but I found one way: I changed function `_merge_flags` into module `portage_config`...

### PR DESCRIPTION
**Pull request by batulin-s**
https://github.com/saltstack/salt/issues/22321

### CODE REVIEW

File: salt/modules/portage_config.py [hunk 1]
```diff
@@ -205,13 +205,26 @@ def _package_conf_ordering(conf, clean=True, keep_backup=False):
                         triplet[0] != path:
                     shutil.rmtree(triplet[0])
 
-
-def _merge_flags(*args):
+def check_accept_keywords(approved, flag):
```
Comment (jfindlay) [inline salt/modules/portage_config.py]: This function needs a docstring.  If it is intended to be externally available, `portage_config.check_accept_keywords`, then it also needs a `.. versionadded:: 2014.7.5`.

**Comment by jfindlay** [comment]
@batulin-s, thanks for fixing this.  There are just a few [lint errors](https://jenkins.saltstack.com/job/salt-pr-lint-n/3915/violations/file/salt/modules/portage_config.py/).

**Comment by batulin-s** [comment]
I'm sorry, I thought that I fixed bugs from https://jenkins.saltstack.com/job/salt-pr-lint-n/3915/violations/file/salt/modules/portage_config.py/
and made new commit (new fix #22321 bug). How to apply this commit?

**Comment by jfindlay** [comment]
I don't know what that error means.  @s0undt3ch or @rallytime, do you know?

**Comment by rallytime** [comment]
`old_flags` should be `None` instead of an empty list. Then inside the function do:
```
if not old_flags:
    old_flags = []
```

**Comment by s0undt3ch** [comment]
As @rallytime said :) :+1:

**Comment by batulin-s** [comment]
@rallytime , Thank You

**Comment by rallytime** [comment]
@batulin-s No problem! Thanks for the fix!

**Pull request merged**